aptos-move/framework/aptos-names/sources/verify.move

```diff
@@ -0,0 +1,32 @@
+module aptos_names::verify {
```

Comment:
nit: Why even make this a separate module and not just add assert_register_domain_signature_verifies to domains.move?

Reply:
max suggested that it's cleaner to have a separate module? since domains.move is mostly for domain-registering code/logic